SPECK/src/main/java/SPEck.java

```diff
@@ -25,28 +25,43 @@
  */
 
 public class SPEck implements Serializable {
+
+    /**
+     * Initializer
+     * @param   scc    <JavaSparkContext> spark context for parallel execution
+     * @param   fileIn  <String> storing the file name of the input dataset to mine the SFSP
+     * @param   fileOut <String> storing the file name of the output (SFSP)
+     * @param   strategy <String> storing the method of sampling the random dataset
```

Comment:
"method" is not a great word to use. I would say "algorithm" or "procedure" . Also, plural dataset*s*.